src/Plugin_Command.php

```diff
@@ -590,6 +591,20 @@ protected function install_from_repo( $slug, $assoc_args ) {
 
 		if ( isset( $assoc_args['version'] ) ) {
 			self::alter_api_response( $api, $assoc_args['version'] );
+		} else {
+			$requires_php = isset( $api->requires_php ) ? $api->requires_php : null;
+			$requires_wp  = isset( $api->requires ) ? $api->requires : null;
+
+			$compatible_php = empty( $requires_php ) || version_compare( PHP_VERSION, $requires_php, '>=' );
+			$compatible_wp  = empty( $requires_wp ) || version_compare( $wp_version, $requires_wp, '>=' );
```

Comment:
Should we mimic `is_wp_version_compatible()` and strip off any suffixes and turn versions like `6.8.0` into `6.8`?

Reply:
Good point I think we should probably do that so that version_compare always works properly. I think to get a minimal mimic of `is_wp_version_compatible()` We could do this:
```
list($core_version) = explode('-', $wp_version);
$core_version = implode('.', array_slice(explode('.', $core_version), 0, 2));
```